### ISSUE TITLE
(feature) Ability to skip logging health end-points

### ISSUE THREAD

**Issue by nimeshsubramanianv2** [thomaspoignant/go-feature-flag#781]
## What
It would be great to have a configuration that will let us disable logging for the health check endpoints. I deployed the relay proxy in k8s, and as an org we generally disable logging for health check end-points.

I can help implement this.

**Comment by thomaspoignant** [comment]
@nimeshsubramanianv2 I agree with you that we don't want to log on those endpoints because it can be super noisy.
I think actually that it should be default behavior and only log if we are running the **relay proxy** in **debug mode**.

**debug mode** is when in the configuration of the relay proxy the field `debug` is set to `true`.

If you want to work on this feel free to open a PR, and please ping me _(here or in slack)_ if you need any help.

**Comment by nimeshsubramanianv2** [comment]
Thank you, I'll try and get you a PR soon. I just wanted to run it by you first before I went about implementing it.

**Comment by thomaspoignant** [comment]
@nimeshsubramanianv2 will you work on it?
If not I can try to do it soon.

**Comment by thomaspoignant** [comment]
@nimeshsubramanianv2 It was a bit more work than I was expecting, but I have opened a PR #790.
If you want to have a look at the PR.

It will be released in the next version.
